### DISCUSSION TITLE
How to use concatenation in NN functions?

### DISCUSSION THREAD

**Discussion by jiaodawuyanzu** [ARM-software/CMSIS_5#1416] [Q&A]
I don't know how to use arm_concatenation_s8_XXX functions, how to use this functions concatenate tensor?

**Comment by felix-johnny** [comment]
@jiaodawuyanzu Please refer to https://github.com/ARM-software/CMSIS_5/blob/963168f61d58ce4e94d211551181a0aed84d68cc/CMSIS/NN/Include/arm_nnfunctions.h#L2115 for pseudo code on the usage. 
  May I ask how you are using the CMSIS-NN APIs? the intention is to use it along with [TFLM](https://github.com/tensorflow/tflite-micro/tree/main/tensorflow/lite/micro/kernels/cmsis_nn) and the list of operators supported are in the link attached. If you are using tflite files along with TFLM's interpreter, then handling of all operators(provided they are supported) will be handled by TFLM's interpreter. That goes for concatenation as well.
